guide/english/certifications/coding-interview-prep/data-structures/size-of-the-set/index.md

```diff
@@ -31,11 +31,41 @@ Use the length attribute on the values of the Set.
 
 ## ![:beginner:](https://forum.freecodecamp.com/images/emoji/emoji_one/beginner.png?v=3 ":beginner:") Basic Code Solution:
 
-    this.size = function()
-    {
+```js
+function Set() {
```

Comment:
Is it a requirement to add the entire function even if the majority of the function is pre-populated?

Reply:
Yes, the CI checks the solution against the tests.

Reply:
It also checks the guide?

Reply:
No, not that I'm aware of. 

Oh, I see. (missed it was the guide file)
IMO, the solution should contain challenge seed to give full context to the reader as I think the solution should be standalone, meaning if someone tries to execute it, it should work.

Reply:
Yes, I agree with @ezioda004.  All guide solutions and solutions added to the challenge file's solution section should be full code.  That way if tests are added later on and do not let the pre-determined solution pass, then there is a problem with the new test added.  It is there for checks and balances.